lib/fastlane/plugin/ddg_apple_automation/helper/git_helper.rb

```diff
@@ -77,6 +77,31 @@ def self.assert_branch_has_changes(release_branch)
 
         changed_files.any?
       end
+
+      def self.latest_release(repo_name, prerelease, platform = nil, github_token)
```

Comment:
I'll probably change the `prerelease` parameter name - what it's really doing it controlling whether the function only looks for a public release, or instead looks for the most recent available release regardless of internal/public status.